### ISSUE TITLE
Feature Request: Support HTML Output for `inspect tree`

### ISSUE THREAD

**Issue by Friendseeker** [sbt/sbt#7981]
Cross Posted from [Discord](https://discordapp.com/channels/632150470000902164/922600050989875282/1323300835035451402)

> is there html renderer for output of inspect tree taskX , like there's one for lib dependencies (dependencyBrowseGraph)?

**Comment by eed3si9n** [comment]
@Friendseeker I don't think we should use "Good first" until we discuss whether such feature is desirable at all, and how this should be implemented, esp given the state of the original `dependencyTree` family of tasks itself in between sbt 1.x and 2.x.

**Comment by Friendseeker** [comment]
> @Friendseeker I don't think we should use "Good first" until we discuss whether such feature is desirable at all, and how this should be implemented, esp given the state of the original `dependencyTree` family of tasks itself in between sbt 1.x and 2.x.

Right. Just removed the label.

**Comment by eed3si9n** [comment]
New ideas should probably be made into a Discussion first, but I got an inquiry from a potential contributor on this, so I'll make a call here.

I personally don't see a value in these render to HTML variants of tasks, and they end up being buggy and also potentially an attack vector since it could launch some Javascript code that's out of date, or difficult to review.

So I'm closing this as won't fix.
